cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -43,16 +40,35 @@ CREATE INDEX IF NOT EXISTS geom_index
 CREATE TABLE IF NOT EXISTS water_systems(
     pws_id varchar(255) NOT NULL,
     lead_connections_count real,
+    geom GEOMETRY(Geometry, 4326),
```

Comment:
Can we add columns for # of total service lines and # of people served by the water system?

That way the client can display estimated LSL % or LSL per capita.

Reply:
Yes that requires editing the water systems lambda so let me do that separately

Reply:
SGTM